src/objects.js

```diff
@@ -1,41 +1,49 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age};
```

Comment:
We usually put spaces between `{` and the property names. Eg `{ name, age, }`. Otherwise, perfect!